app/src/main/java/org/simple/clinic/summary/PatientSummaryViewRenderer.kt

```diff
@@ -50,6 +50,10 @@ class PatientSummaryViewRenderer(
   }
 
   private fun renderClinicalDecisionBasedOnAppointment(model: PatientSummaryModel) {
+    //We don't show CDSS alert when statin alert is shown
```

Comment:
Since you named the variable properly, the code is self explanatory, we don't need this comment. WDYT?